doc/v2/howto/cluster/multi_cluster/k8s_distributed_en.md

```diff
@@ -1,3 +1,365 @@
-# Kubernetes Distributed
+# Kubernetes Distributed Training
 
-TBD
+We introduced how to create a PaddlePaddle Job with a single node on Kuberentes in the
+previous document.
+In this article, we will introduce how to craete a PaddlePaddle job with multiple nodes
+on Kubernetes cluster.
+
+## Overall Architecture
+
+Before creating a training job, the users need to deploy the Python scripts and
+training data which have already been sliced on the precast path in the distributed file
+system(We can use the different type of Kuberentes Volumes to mount different distributed
+file system). Before start training, The program would copy the training data into the
+Container and also save the models at the same path during training. The global architecture
+is as follows:
+
+![PaddlePaddle on Kubernetes Architecture](src/k8s-paddle-arch.png)
+
+The above figure describes a distributed training architecture which contains 3 nodes, each 
+Pod would mount a folder of the distributed file system to save training data and models
+by Kubernetes Volume. Kubernetes created 3 Pod for this training phase and scheduled these on
+3 nodes, each Pod has a PaddlePaddle container. After the containers have been created,
+PaddlePaddle would start up the communication between PServer and Trainer and read training
+data for this training job.
+
+As the description above, we can start up a PaddlePaddle distributed training job on a ready
+Kubernetes cluster as the following steps:
+
+1. [Build PaddlePaddle Docker Image](#Build a Docker Image)
+1. [Split training data and upload to the distributed file system](#Upload Training Data)
+1. [Edit a YAML file and create a Kubernetes Job](#Create a Job)
+1. [Check the output](#Check The Output)
+
+We will introduce these steps as follows:
+
+### Build a Docker Image
+
+PaddlePaddle Docker Image needs to support the runtime environment of `Paddle PServer` and
+`Paddle Trainer` process and this Docker Image has the two import features:
+
+- Copy the training data into the container.
+- Generate the start arguments of `Paddle PServer` and `Paddle Training` process.
+
+Because of the official Docker Image `paddlepaddle/paddle:latest` has already included the
```

Comment:
Since the paddlepaddle official docker image already has the runtimes we need, we'll take it as the base image and pack some additional scripts for the processes mentioned above to build our training image. for more detail, please find from the following link: